efficient-eagles/early_internet/main/models.py

```diff
@@ -21,6 +21,10 @@ class CustomUser(AbstractUser):
     def __str__(self):
         return self.username
 
+    @property
```

Comment:
What is the point ?

Reply:
It was so that in the admin page, the username would be viewed as a string and not something like, "object (1)", but I guess it's really not needed

Reply:
Im asking about 24, 25 lije8

Reply:
oh, right, yeah that can be removed